src/solver/level_initializer/hybrid_level_initializer.h

```diff
@@ -35,7 +35,9 @@ namespace solver
         static constexpr auto interp_order = GridLayoutT::interp_order;
 
 
-        PHARE::core::Ohm<GridLayoutT> ohm_;
+        PHARE::core::Ohm<GridLayoutT>
+            ohm_(dict); // where the fuck should this dict come from ? should we make an explicit
```

Comment:
either 

```
explicit HybridLevelInitializer(PHARE::core::Ohm<GridLayoutT>&& ohm) : ohm_{ohm} {
```

or

```
explicit HybridLevelInitializer(PHARE::initializer::PHAREDict dict) : ohm_{dict} {
```

I suppose, the former is minimal, the latter is more extensible should "ampere_" need parameters later for example